src/Dijkstra/AST/Examples/PaperIntro.agda

```diff
@@ -0,0 +1,165 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
```

Comment:
It seems like the framework helps.
But it certainly seems one needs to be well-versed in Agda+emacs to know when/what to do each step.